components/post_view/channel_intro_message/add_members_button.tsx

```diff
@@ -66,37 +64,30 @@ const AddMembersButton: React.FC<AddMembersButtonProps> = ({totalUsers, usersLim
     );
 };
 
-const LessThanMaxFreeUsers = ({setHeader, createBoard}: {setHeader: React.ReactNode; createBoard: React.ReactNode}) => {
+const LessThanMaxFreeUsers = ({createBoard}: {setHeader: React.ReactNode; createBoard: React.ReactNode}) => {
     const {formatMessage} = useIntl();
 
     return (
         <>
             {createBoard}
-            {setHeader}
             <div className='LessThanMaxFreeUsers'>
-                <EmptyStateThemeableSvg
-                    width={128}
-                    height={113}
-                />
                 <div className='titleAndButton'>
-                    <FormattedMessage
-                        id='intro_messages.inviteOthersToWorkspace.title'
-                        defaultMessage='Let’s add some people to the workspace!'
-                    />
                     <ToggleModalButton
                         ariaLabel={localizeMessage('intro_messages.inviteOthers', 'Invite others to the workspace')}
                         id='introTextInvite'
-                        className='intro-links color--link cursor--pointer'
+                        className='intro-links padding-extra color--link cursor--pointer'
                         modalId={ModalIdentifiers.INVITATION}
                         dialogType={InvitationModal}
                     >
-                        <i
-                            className='icon-email-plus-outline'
+                        <EmailOutlineIcon
+                            size={14}
+                            css={{marginRight: 7}}
```

Comment:
instead of passing in an arbitrary value each time could you please adjust the styling for the `intro-links` class and add `gap: 8px;` to it (it is already using `display: flex;` and we are aiming to use a `4px` grid, so `8px` would be the closest possible value for that)